components/mysql/actions/delete-row/delete-row.mjs

```diff
@@ -0,0 +1,48 @@
+import mysql from "../../mysql.app.mjs";
+
+export default {
+  key: "mysql-delete-row",
+  name: "Delete Row",
+  description: "Delete an existing row. [See the docs here](https://dev.mysql.com/doc/refman/8.0/en/delete.html)",
+  type: "action",
+  version: "0.0.1",
+  methods: {
+    async deleteRows({
```

Comment:
Since the the `deleteRows` method is just called here, and the `run` function is pretty small. I would like to suggest to move the code to the `run`function